org.OpenT2T.Onboarding.VeraHub/node/README.md

```diff
@@ -0,0 +1,9 @@
+# Vera Hub Onboarding
+Sample [Vera Hub](http://www.getvera.com/) onboarding module for Open Translators to Things. May be used to discover devices connected to a Vera Hub per the 
+[org.OpenT2T.Onboarding.VeraHub.xml](https://github.com/openT2T/onboarding/blob/master/org.OpenT2T.Onboarding.VeraHub/org.OpenT2T.Onboarding.VeraHub.xml) schema.
+
+See schema definition for inputs to the onboarding method, and outputs via success and error callbacks.
+
+## Sample usage (via test.js script)
+Lights: node test -f 'Dimmable'
```

Comment:
great. thanks.

---
In reply to: [69950373](https://github.com/openT2T/onboarding/pull/5#discussion_r69950373) [](ancestors = 69950373)